vos/docker/README.rst

```diff
@@ -0,0 +1,41 @@
+Docker builds
+=============
+
+Versions
+~~~~~~~~
+
+Supported Python versions are:
+
+ - 2.7
+ - 3.6
+ - 3.7
+ - 3.8
+
+
+There are Dockerfiles for each version in the docker/${version} folders.
+
+Integration Tests
+~~~~~~~~~~~~~~~~~
+
+Only 2.7 is supported for Integration Tests.
```

Comment:
Why? We are trying to discourage users from using 2.7...

Reply:
Absolutely!  This message is a little misleading.  It means to convey that the integration tests use 2.7 syntax which is incompatible with 3.  Nobody going forward should be using 2.7, so ultimately we do want to discourage it.